pyodide-build/pyodide_build/cli/xbuildenv.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+import typer
+
+from ..install_xbuildenv import install
+
+app = typer.Typer(hidden=True)
+
+
+@app.callback()
+def callback():
+    """
+    Create or install cross build environment
+    """
+
+
+@app.command("install")  # type: ignore[misc]
+def _install(
```

Comment:
Is this hidden from `pyodide --help` or just the docs?

Reply:
It is hidden from both of help message and docs.